packages/loaders/src/Dots.js

```diff
@@ -62,9 +62,9 @@ export default function Dots({
       {...other}
     >
       <g fill="currentColor">
-        <DotsOneCircle duration={duration} ref={dotOne} />
-        <DotsTwoCircle duration={duration} ref={dotTwo} />
-        <DotsThreeCircle duration={duration} ref={dotThree} />
+        <DotsOneCircle duration={duration} ref={noAnimatedSVGSupport ? dotOne : null} />
+        <DotsTwoCircle duration={duration} ref={noAnimatedSVGSupport ? dotTwo : null} />
+        <DotsThreeCircle duration={duration} ref={noAnimatedSVGSupport ? dotThree : null} />
```

Comment:
Why are is conditionally apply the refs here necessary?

Reply:
I don't think it's worth checking if the ref is needed or not, theres little cost in applying a ref even if it isn't used.